File: StudyHard/StudyHard/ViewController.m
//
//  ViewController.m
//  StudyHard
//
//  Created by waitwalker on 2020/12/22.
//

#import "ViewController.h"
#import <DeviceHelper/DeviceManager.h>


@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    DeviceType type = [DeviceManager sharedManager].deviceType;
    NSLog(@"type:%ld",type);
}


@end

File: KeyboardDemo/KeyboardDemo/ViewController.swift
//
//  ViewController.swift
//  KeyboardDemo
//
//  Created by waitwalker on 2021/3/31.
//

import UIKit
import KeyboardKit

class ViewController: KeyboardInputViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        let textField = UITextField(frame: CGRect(x: 100, y: 100, width: 200, height: 100))
        textField.backgroundColor = .red
        self.view.addSubview(textField)
        
    }


}

